src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -92,7 +92,7 @@ export function getChmInfo(genomeBuild:string) {
     if (!referenceGenome || referenceGenome === "") {
         referenceGenome = DEFAULT_GENOME_BUILD;
     }
-    const genomeSize = referenceGenomeSizes[genomeBuild];
+    const genomeSize = referenceGenomeSizes[referenceGenome];
```

Comment:
do we have a sense why this working for other patients?  seems like we were using the key instead of the value

Reply:
for other patients `ncbiBuild` of [first mutation](https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/genomicOverview/Tracks.tsx#L58) is in one of `referenceGenomeSizes` keys

Reply:
hi @alisman , @kalletlak , my mistake, should be **_referenceGenome_**. Thanks!

Reply:
thanks @khzhu !!!